test/unit_samples.jl

```diff
@@ -0,0 +1,31 @@
+# using Pkg
+# Pkg.activate("LazySets")
+# using Test
+
+using LazySets
+using Distributions
+using Polyhedra
```

Comment:
Eventually remove these lines above (they are not needed).

Reply:
There are some commented lines on top that can also be removed.

Reply:
Will do that. I left them there because I am still not able to run the `runtest.jl` file due to problem with `CDDLib` on Windows.